extensions/wikia/Forum/DiscussionsRedirectHelper.class.php

```diff
@@ -0,0 +1,77 @@
+<?
+class DiscussionsRedirectHelper {
+	/**
+	 * @return bool
+	 */
+	public static function areForumsArchivedAndDiscussionsEnabled() {
+		global $wgArchiveWikiForums, $wgEnableDiscussions;
+
+		return $wgArchiveWikiForums && $wgEnableDiscussions;
```

Comment:
If think this should actually be `empty( $wgEnableForumExt ) && $wgEnableDiscussions`. `$wgArchiveWikiForums` is used to put old wiki forums (/wiki/Forum:Index) into archive mode, which is slightly different than disabled threaded forums (/wiki/Special:Forum).